src/plugins/xerces/xerces.cpp

```diff
@@ -0,0 +1,164 @@
+/**
+ * @file this file contains the entry point to the plugin as a gateway between c and c++
+ *
+ * @brief Plugin enables storage to xml files via the Xerces library
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ */
+
+#include "xerces.hpp"
+#include "deserializer.hpp"
+#include "serializer.hpp"
+#include "util.hpp"
+
+#include <stdio.h>
+
+#include <xercesc/dom/DOM.hpp>
+#include <xercesc/util/OutOfMemoryException.hpp>
+#include <xercesc/util/PlatformUtils.hpp>
+
+using namespace ckdb; // required for kdberrors.h
+
+#include <kdberrors.h>
+#include <kdbhelper.h>
+
+XERCES_CPP_NAMESPACE_USE
+
+int elektraXercesOpen (Plugin * handle, Key * errorKey)
+{
+	XMLPlatformUtils::Initialize ();
+	return 1;
+}
+
+int elektraXercesClose (Plugin * handle, Key * errorKey)
+{
+	XMLPlatformUtils::Terminate ();
```

Comment:
According to https://xerces.apache.org/xerces-c/faq-parse-3.html#faq-4 it is. Only restriction is that it should be from the same thread (i haven't really cared about that for now, i guess thats usually the case anyway?)